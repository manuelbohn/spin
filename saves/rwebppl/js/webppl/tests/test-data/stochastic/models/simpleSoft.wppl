var model = function() {
  var x = flip();
  var y = flip();
  factor((x | y) ? 0 : -2);
  return x;
};
