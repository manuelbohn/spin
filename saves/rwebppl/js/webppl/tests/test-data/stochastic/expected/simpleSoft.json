{
  "hist": {
    "true": 0.6379,
    "false": 0.3621
  }
}
